### ISSUE TITLE
Error: No such option: --disable-telemetry Did you mean --disable-optional-telemetry?

### ISSUE THREAD

**Issue by AlirezaSoltaniJazi** [Lucas-C/pre-commit-hooks-safety#48]
Hi,
I hope you're doing well.
I'm using this config:
```
-   repo: https://github.com/Lucas-C/pre-commit-hooks-safety
    rev: v1.3.1
    hooks:
    -   id: python-safety-dependencies-check
        name: Check dependencies for vulnerability
        files: requirements*
        args: ["--disable-telemetry"]
```
but I got this error:
```
Error: No such option: --disable-telemetry Did you mean --disable-optional-telemetry?
```

**Comment by Lucas-C** [comment]
Well, have you tried replacing `--disable-telemetry` by `--disable-optional-telemetry`? 😅

**Comment by massood** [comment]
Hi @Lucas-C 

I have the same issue. My `.pre-commit-config.yaml` is the same as OPs.

When I use `args: ["--disable-optional-telemetry"]`, I get the following error:
```
$ pre-commit run python-safety-dependencies-check -a
Check dependencies for vulnerability.....................................Failed
- hook id: python-safety-dependencies-check
- exit code: 2

Usage: safety check [OPTIONS]
Try 'safety check --help' for help.

Error: No such option: --disable-optional-telemetry
```
which is expected, because `pre-commit-hooks-safety` does not move this option before the `check` sub-command.

More importantly, when I run the `safety` itself with `--disable-telemetry`, everything works fine:
```
safety --disable-telemetry check -r requirements.txt 

...

 REPORT 

  Safety v2.3.5 is scanning for Vulnerabilities...
  Scanning dependencies in your files:

  -> requirements.txt

  Using non-commercial database
  Found and scanned 93 packages
  Timestamp 2024-01-20 13:30:33
  0 vulnerabilities found
  0 vulnerabilities ignored

...

 No known security vulnerabilities found. 
```

But when I run `pre-commit-hooks-safety` with it, I get the following error:
```
$ pre-commit run python-safety-dependencies-check -a
Check dependencies for vulnerability.....................................Failed
- hook id: python-safety-dependencies-check
- exit code: 2

Usage: safety [OPTIONS] COMMAND [ARGS]...
Try 'safety --help' for help.

Error: No such option: --disable-telemetry Did you mean --disable-optional-telemetry?
```

We haven't had this issue until recently, although our source code haven't been changed.

Also, I cannot re-produce it on my local development machine. But it happens on our production clean environment with the exact same `pip freeze`d packages as my local machine.

**Comment by AlirezaSoltaniJazi** [comment]
> Hi!
> 
> Well, have you tried replacing `--disable-telemetry` by `--disable-optional-telemetry`? 😅

Yes, I have. I changed it and after that I got this error:
```
Error: No such option: --disable-optional-telemetry
````

**Comment by thorroed** [comment]
It seems that that argument has changed upstream and it's position needs to be before the actual command. 
does this PR resolve it for you?
#49

**Comment by Lucas-C** [comment]
This has been fixed by @thorroed in PR #49 and has just been released in v1.3.3:
https://github.com/Lucas-C/pre-commit-hooks-safety/releases/tag/v1.3.3
